### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by sooyitao**
- Updated theme to a lighter colour palette
- Updated PersonCard to be more compact
- Updated fonts and font size
- Added icons
- Added command summary to help window

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -50,8 +58,14 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
+        Image phone = new Image(getClass().getResourceAsStream("/images/phone_icon.png"));
```
Comment (george-yeo) [inline src/main/java/seedu/address/ui/PersonCard.java:61]: Image path name "/images/phone_icon.png" could be made into a final static variable for better organisation.

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 2]
```diff
@@ -50,8 +58,14 @@ public PersonCard(Person person, int displayedIndex) {
...
+        phoneIcon.setImage(phone);
         address.setText(person.getAddress().getValueForUI());
+        Image address = new Image(getClass().getResourceAsStream("/images/address_icon.png"));
+        addressIcon.setImage(address);
         email.setText(person.getEmail().getValueForUI());
+        Image email = new Image(getClass().getResourceAsStream("/images/email_icon.png"));
```
Comment (george-yeo) [inline src/main/java/seedu/address/ui/PersonCard.java:67]: The other path names here too.

**Review by george-yeo — APPROVED** [review]
LGTM!

**Review by ty4g1 — COMMENTED** [review]
A small issue, when user clicks on the contact entry, it changes color to a dark gray background, I think this is a missed change? It makes the text hard to read, so I'd suggest changing that selected background to a lighter shade of gray. Otherwise the UI looks great!

**Pull request merged**